src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewAdapter.cs

```diff
@@ -623,6 +623,10 @@ internal List<Cell> GetCellsFromPosition(int position, int take)
 		void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
 		{
 			OnDataChanged();
+			if(e.Action == NotifyCollectionChangedAction.Reset)
+			{
+				_realListView.SmoothScrollToPosition(0);
```

Comment:
With this change, we align the behavior between all platforms?

Reply:
> With this change, we align the behavior between all platforms?

Yes, on the other platforms, the scroll position was automatically set to 0, so this change ensures consistent behavior across all platforms.